uwuno/src/main/java/com/learning/uwuno/controllers/roomController.java

```diff
@@ -55,10 +56,19 @@
     // PUTS
     @PutMapping(value = "rooms/{uid}")
     public ResponseEntity<room> updateRoom(@PathVariable String uid, @RequestBody String json) {
-            parser parser = new parser(json);
+        parser parser = new parser(json);
```

Comment:
Does the POST not allow use to set game mode and stuff?

Reply:
No, it does not. The setting won't be set with a POST request. Our front-end design is: lobby (create room) -> settings (where we add players, set game settings) -> in-game.

I added blankCard as a key for gameSettings. So we can move blankCards away from POST.